jobrunner/models.py

```diff
@@ -66,6 +66,7 @@ class StatusCode(Enum):
     CANCELLED_BY_USER = "cancelled_by_user"
     UNMATCHED_PATTERNS = "unmatched_patterns"
     INTERNAL_ERROR = "internal_error"
+    KILLED_BY_ADMIN = "Killed by admin"
```

Comment:
oh nice. Can you add this is another final state that all jobs can go to in the mermaid diagram in `DEVELOPERS.md`? It should be c/p of INTERNAL_ERROR bits, basically